app/css/openmrs-owa-orderentry.css

```diff
@@ -482,39 +482,57 @@ a.button:hover {
     padding: 5px 10px !important;
 }
 
-.draft-list-container .draft-list{
+.draft-list-container .draft-item{
     display: block;
     background: #EBEBEB;
-    padding: 10px 10px;
     margin: 5px 0px;
     display: flex;
     align-items: center;
     position: relative;
     border-radius: 5px;
 }
 
-.draft-list-container .draft-list:hover{
+.draft-list-container .draft-item > span:hover{
     background: #D3D3D3;
     transition: all 200ms ease-in-out;
     -webkit-transition: all 200ms ease-in-out;
     -moz-transition: all 200ms ease-in-out;
 }
 
+.right:hover {
+    border-top-right-radius: 5px;
+    border-bottom-right-radius: 5px
+}
+
+.draft-name {
+    padding: 10px 10px;
+    width: 60%;
+}
+
 .stay-right{
     position: absolute;
     right: 10px;
 }
 
+.push-right {
```

Comment:
This class does not seem to be used, and also has the `!mportant` tag, please let's remove it.

Reply:
done